From 13690179afc776f53dad4478b3bc88f6041d95e2 Mon Sep 17 00:00:00 2001
From: Akshaya Bhat <akshayab@qti.qualcomm.com>
Date: Mon, 29 Apr 2024 22:25:56 -0700
Subject: [PATCH] [PATCH] Add 32 bit busybox substitution paths

---
 config/file_contexts.subs_dist | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config/file_contexts.subs_dist b/config/file_contexts.subs_dist
index f80499ebf..d336039d4 100644
--- a/config/file_contexts.subs_dist
+++ b/config/file_contexts.subs_dist
@@ -46,6 +46,10 @@
 /usr/lib/busybox/sbin /usr/sbin
 /usr/lib/busybox/usr /usr
 
+/usr/lib/lib32-busybox/bin /usr/bin
+/usr/lib/lib32-busybox/sbin /usr/sbin
+/usr/lib/lib32-busybox/usr /usr
+
 # The genhomedircon.py will expand /root home directory to /home/root
 # Add an aliase for it
 /root /home/root
-- 
2.17.1

